test/specs/map/PreviewFeaturesService.spec.js

```diff
@@ -155,6 +160,8 @@ describe('ga_previewfeatures_service', function() {
       });
 
       it('loads then adds new features from their ids', function(done) {
+        var mock = gaLayersMock.expects('getLayer').withArgs('somelayer').returns(layerBodType);
```

Comment:
No need of a mock,  a stub is enough.

`sinon.stub(gaLayers, 'getLayer').returns({});`